styles.css

```diff
@@ -476,5 +488,240 @@ input:focus {
 .home-indicator {
   width: 134px;
   border-radius: 10px;
-  margin-top: 100px;
+  margin-top: 30px;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  html {
+    box-sizing: border-box;
+  }
+
+  .container {
+    width: auto;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .head {
+    width: 100%;
+    background-color: #fff;
+    position: fixed;
+  }
+
+  .navbar {
+    height: 72px;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: space-around;
+    width: 250px;
+  }
+
+  /* headline section */
+  .headline {
+    height: 100vh;
+    background-image: url(./src/assets/img/desktop-head-back.svg);
+    background-size: contain;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .head-wraper {
+    margin-top: -120px;
+  }
+
+  .head-primary {
+    width: 920px;
+    height: 122px;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .head-intro {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    align-items: center;
+  }
```

Comment:
- [ ] The same comment here for the width kindly don't fix it, please know that your design needs to be flexible, kindly fix this as well. 😁